### PR TITLE
Assignment features

### PR DESCRIPTION
**Pull request by irfan-iiitr**
**Tasks Done**
1. Get assignments by LevelID
2. Endpoints for project submission
3.  Verify submission

### CODE REVIEW

**Comment by irfan-iiitr** [comment]
@aialok Please review this.

File: src/controllers/assignment.controller.ts [hunk 1]
```diff
@@ -109,10 +109,76 @@ const deleteAssignment = async (req: Request, res: Response) => {
   }
```
Comment (aialok) [inline src/controllers/assignment.controller.ts:109]: LGTM

File: src/repositories/assignment.repository.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { level } from "winston";
```
Comment (aialok) [inline src/repositories/assignment.repository.ts:1]: LGTM

**Review by aialok — CHANGES_REQUESTED** [review]
LGTM

**Pull request merged**